source/dotnet/wholesale-api/WebApi.UnitTests/Infrastructure/EventPublishers/IntegrationEventTypeMapperTests.cs

```diff
@@ -40,26 +43,42 @@ public void GetEventName_WhenEventType_ReturnsEventName(IntegrationEventTypeMapp
 
     [Theory]
     [AutoData]
-    public void GetEventType_WhenEventName_ReturnsEventType(IntegrationEventTypeMapper sut)
+    public void ThrowsException_WhenGettingNotExisting(IntegrationEventTypeMapper sut)
     {
         // Arrange
-        var type = typeof(string);
-
-        // Act
-        sut.Add("string", type);
+        var eventType = typeof(CalculationResultCompleted);
 
-        // Assert
-        Assert.Equal(1, sut.Count());
+        // Act & Assert
+        Assert.Throws<KeyNotFoundException>(() => sut.GetMessageType(eventType));
     }
 
     [Theory]
     [AutoData]
-    public void ThrowsException_WhenGettingNotExisting(IntegrationEventTypeMapper sut)
+    public void OnInitialization_WhenAddingExistingEventType_ThrowsException(
```

Comment:
I'm not sure but I think we normally name it like this (or perhaps it's just my personal preference). Feel free to decide what you prefer.
```suggestion
    public void Ctor_WhenAddingExistingEventType_ThrowsException(
```

Reply:
![image](https://user-images.githubusercontent.com/11583438/228448580-a4349924-ae79-4981-91fc-45292b2f8a84.png)